lib/tasks/publish_finders.rake

```diff
@@ -8,4 +8,58 @@ namespace :finders do
       PublishFinder.call(content_item)
     end
   end
+
+  desc "Temporary task to unpublish non-English finders - this can be removed after running in production"
+  task temp_unpublish_non_english_finders: :environment do
+    locales = Locale.non_english.map(&:code)
+
+    locales.each do |locale|
+      puts "Publishing content item for /government/announcements.#{locale}"
+      publish_finder_for_locale("88936763-df8a-441f-8b96-9ea0dc0758a1", "/government/announcements.#{locale}", locale)
```

Comment:
I'd be tempted to pull out these content ids and name them as variables so they're a bit less magic. 
But again, given temp, not super fussed

Reply:
That's a good idea, I'll do that now to make sure we're referring to the same content ID continuously.